corehq/toggles.py

```diff
@@ -367,6 +367,14 @@ def toggle_values_by_name(username=None, domain=None):
     [NAMESPACE_DOMAIN, NAMESPACE_USER]
 )
 
+VELLUM_BETA = PredictablyRandomToggle(
+    'vellum_beta',
+    'Use Vellum beta version',
+    TAG_ONE_OFF,
+    [NAMESPACE_DOMAIN],
+    randomness=0.1
```

Comment:
Curious how you decided to do predictably random instead of all domains with the other two flags set (I think "rich text" and "experimental UI") plus a few hand-picked others? I'm not against this, just curious. Is it easy to tell which domains are chosen/affected by this?